src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+
+This exists to keep people from wearing them 24/7 as people did with modsuits, using less abilities would drain charge slower, allowing people to disable things when unneeded to conserve power, or just never use them for extended spacesuit time. The same applies to antagonists, though your blood-red hardsuit has a much higher energy capacity than any of the crew's hardsuits, you'll still have to charge it at some point, and when that time comes you'll be extremely vulnerable. No more syndies buying a jetpack and a blood-red and just flying around the station for the entire round shooting people.
+
+Running out of charge would not prevent the suit from protecting you from space, but it would give you a major movement speed decrease (think 25%-75% depending on the suit) and entirely prevent you from using ANY abilities. Stored items will be locked inside, magboots will be turned off, and additional armor shields will be disabled. Engineers could still wrap up repairs after running out of power if they have other equipment rather than relying on their suits, and security could finish dragging an apprehended suspect back to their department, just very slowly.
+
+## Suit Abilities
+
+Specific hardsuits would have specific abilities that could be seen in SS13 modsuits, but these abilities would be hard-locked to specific suits, rather than being placed on any suit with a module. The abilities could include...
+
+- **A basic flashlight on the helmet/suit.**
+A small light with a low energy cost that lets you see. You could keep it off in a lit area, or if you want to conserve suit charge. Carrying a seclight or a normal flashlight is also viable.
+- **Engi hardsuits giving welding protection.**
+Maybe you want to wear sunglasses or a medhud or just don't like how engi goggles look on your character, well it doesn't come free, since having this turned on would also cost suit charge. An efficient player could turn it on/off whenever they're welding, but that runs the risk of forgetting to turn it on and damaging your eyes.
+- **Engi hardsuits having built-in magboots.**
+Perhaps you didn't grab magboots before gravity went out or stole an engi suit and don't have access to them at all, either way, you still need to use suit charge to power them.
+- **Atmos hardsuits having thermal regulators that when disabled don't protect you from extreme heat.**
+A similar situation to the welding goggles ability. You can either leave it turned on at all times while wearing the suit, or you can turn it on/off while you're dealing with a recent plasma leak and subsequent fire. Conserving suit charge at slight risk of forgetting and causing your death.
```

Comment:
I would like to avoid inherent noobtraps. I like the idea, but right *now* at least, getting exposed to 20,000C gasses for a microsecond just kills you no matter how fast you react, and for my poor moff friends, it also just fucking ashes you. 

I'd like to see fire/absurdly dangerous Atmos protection possibly scale compared to how dangerous the Atmos is. 

Think about it this way: a 5,000C atmosphere can be stomached perfectly fine. It would have little to no power draw, because 5,000C is not really outside a realistic realm for material resistance for an advanced hardsuit engineered for the conditions...

But a 20,000C environment? The suit is working HARD to keep the meaty organic inside cool as possible compared to the absolute hell outside. It would use a lot of power and Atmos techs would be encouraged to fix a problem as fast as possible rather than "this room is now 20,000C, have fun trying to catch me" and casually sitting in it all day.